ubuntu-vm-oms-extension/README.md

```diff
@@ -0,0 +1,7 @@
+# Deployment of a Ubuntu VM with OMS Extension
```

Comment:
Please rename the folder to 201-oms-extension-ubuntu-vm since this is an example of platform capability